auditbeat/module/auditd/_meta/fields.yml

```diff
@@ -2,51 +2,158 @@
+    - name: uid
+      type: keyword
+      description: user ID
+    - name: euid
+      type: keyword
+      description: effective user ID
+    - name: fsuid
+      type: keyword
+      description: file system user ID
+    - name: suid
+      type: keyword
+      description: sent user ID
+    - name: gid
+      type: keyword
+      description: group ID
+    - name: egid
+      type: keyword
+      description: effective group ID
+    - name: sgid
+      type: keyword
+      description: set group ID
+    - name: fsgid
+      type: keyword
+      description: file system group ID
+    - name: selinux
       type: group
+      description: The SELinux identity of the actor.
       fields:
-      - name: action
+      - name: user
+        type: keyword
+        description: account submitted for authentication
+      - name: role
+        type: keyword
+        description: user's SELinux role
+      - name: domain
         type: keyword
-        example: logged-in
-        description: A description of the action taken by the user.
+        description: The actor's SELinux domain or type.
+      - name: level
+        type: keyword
+        example: s0
+        description: The actor's SELinux level.
+      - name: category
+        type: keyword
+        description: The actor's SELinux category or compartments.
+
+  - name: process
+    type: group
+    description: Process attributes.
+    fields:
+    - name: pid
+      type: keyword
+      description: Process ID.
+    - name: ppid
+      type: keyword
+      description: Parent process ID.
+    - name: name
+      type: keyword
+      description: Process name (comm).
+    - name: title
+      type: keyword
+      description: Process title or command line parameters (proctitle).
+    - name: exe
+      type: keyword
+      description: Absolute path of the executable.
+    - name: cwd
+      type: keyword
+      description: The current working directory.
+    - name: args
+      type: keyword
+      description: The process arguments as a list.
+
+  - name: source
+    type: group
+    description: Source that triggered the event.
+    fields:
+    - name: ip
+      type: ip
+      description: The remote address.
+    - name: port
+      type: keyword
+      description: The port number.
+    - name: hostname
+      type: keyword
+      description: Hostname of the source.
+    - name: path
+      type: keyword
+      description: This is the path associated with a unix socket.
+
+  - name: destination
+    type: group
+    description: Destination address that triggered the event.
+    fields:
+    - name: ip
+      type: ip
+      description: The remote address.
+    - name: port
+      type: keyword
+      description: The port number.
+    - name: hostname
+      type: keyword
+      description: Hostname of the source.
+    - name: path
+      type: keyword
+      description: This is the path associated with a unix socket.
+
+  - name: network.direction
```

Comment:
Fingers cross that this is not changing ;-)

Reply:
I think we need to standardize on the enum values.

Reply:
+1 on having a recommend standard.